jsonschema-core.xml

```diff
@@ -2827,32 +2821,62 @@ https://example.com/schemas/common#/$defs/count/minimum
+                <section title="Errors">
                     <t>
-                        The error or annotation that is produced by the validation.
+                        Any errors produced by the validation.  This property MUST NOT
+                        be included if the validation was successful.  The value
+                        for this property MUST be an object where the keys are the names of
+                        keywords and the values are the error message produced by the
+                        associated keyword.  If the subschema itself is producing the
+                        error, that error MUST be listed with an empty string key.
                     </t>
                     <t>
-                        For errors, the specific wording for the message is not defined by this
+                        The specific wording for the message is not defined by this
                         specification.  Implementations will need to provide this.
                     </t>
                     <t>
-                        For annotations, each keyword that produces an annotation specifies its
-                        format.  By default, it is the keyword's value.
+                        The JSON key for this information is "errors".
+                    </t>
+                </section>
+
+                <section title="Annotations">
+                    <t>
+                        Any annotations produced by the validation.  This property MUST NOT
+                        be included if the validation was unsuccessful.  The value
+                        for this property MUST be an object where the keys are the names of
+                        keywords and the values are the annotations produced by the
+                        associated keyword.
                     </t>
                     <t>
-                        The JSON key for failed validations is "error"; for successful validations
-                        it is "annotation".
+                        Each keyword defines its own annotation data type (e.g. "properties"
+                        produces a list of keywords, whereas "$title" produces a string).
+                    </t>
+                    <t>
+                        The JSON key for this information is "annotations".
                     </t>
                 </section>
 
                 <section title="Nested Results">
                     <t>
-                        For "basic", this property will appear only at the root node and will hold
-                        all errors or annotations in a list.
+                        Nested results are generated from keywords which create a new dynamic
+                        scope by applying a subschema to the instance or a child of the instance.
+                        Keywords which have multiple subschemas (e.g. "anyOf") will generally
+                        generate an output unit for each subschema.  In order to accommodate
```

Comment:
This is an excellent use of "subschema", just to clarify that I don't mean eliminating the term everywhere :-)